quinn-proto/src/connection/mod.rs

```diff
@@ -2113,6 +2116,12 @@ impl Connection {
             }
         }
 
+        if !self.resumption_tickets_received && self.crypto.resumption_tickets_received() > Some(0)
```

Comment:
Is the number of tickets received guaranteed to change at most once in the lifetime of a connection? If not, should we raise an event every time it changes?